pkg/volume/gce_pd/gce_util.go

```diff
@@ -146,7 +146,7 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (volum
 		return "", 0, err
 	}
 
-	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB))
+	err = cloud.CreateDisk(name, zone.FailureDomain, int64(requestGB), *c.options.CloudTags)
```

Comment:
A separate PR for the others makes sense, but why not this PR for this name?  Is getting the PV name onto the provisioner `c` object more invasive than I assume?